smtp/client.js

```diff
@@ -85,7 +85,7 @@ Client.prototype =
       if(!(msg instanceof message.Message) 
           && msg.from 
           && (msg.to || msg.cc || msg.bcc)
-          && (msg.text || this._containsInlinedHtml(msg.attachment)))
+          && ((msg.text || '' != null) || this._containsInlinedHtml(msg.attachment)))
```

Comment:
right. if you took out the double pipe and just checked for undefined, it should then pass, as long as the user has set `msg.text` to something